src/rebar3-nix-bootstrap

```diff
@@ -89,34 +89,47 @@ bootstrap_libs(#data{erl_libs = ErlLibs}) ->
                             gather_directory_contents(Path) ++ Acc
                     end, [], Paths),
     lists:foreach(fun (Path) ->
-                          make_symlink(Path, Target)
+                          link_app(Path, Target)
                   end, CopiableFiles).
 
--spec gather_directory_contents(string()) -> [string()].
+-spec gather_dependency(string()) -> [{string(), string()}].
+gather_dependency(Path) ->
+    FullLibrary = filename:join(Path, "lib/erlang/lib/"),
+    case filelib:is_dir(FullLibrary) of
+        true ->
+            gather_directory_contents(FullLibrary);
+        false ->
+            [raw_hex(Path)]
+    end.
+
+-spec raw_hex(string()) -> {string(), string()}.
+raw_hex(Path) ->
+    [_, Name] = re:split(Path, "-hex-source-"),
```

Comment:
Can you also add a comment to the fetchHex nix expression that this magic string is used in here?